history/management/commands/run_bot.py

```diff
@@ -288,6 +289,10 @@ def won(message,opponentname,gamename):
             #send response
             gifurl = get_gif('winorloss')
             message.send("#win recorded \n {}".format(gifurl))
```

Comment:
I would agree with that; the channel can get a bit painful to look at, with all the moving and the pictures and whatnot.